docs/intro.md

```diff
@@ -45,22 +48,23 @@ In order to secure the network, developers must bootstrap a sufficiently secure
 Also, in a potential future with millions of app-chains, it’s highly unlikely all of those chains will be able to sustainably attract a sufficiently secure and decentralized validator set.
 
 ## Why Rollkit?
+Rollkit solves the challenges encountered during the deployment of a smart contract or a new layer 1, by minimizing these tradeoffs through the implementation of rollups.
```

Comment:
how does Rollkit solve the challenge encountered during the deployment of a smart contract? I don't know this is true, or what Rollkit is for?

Reply:
problem statement :
Deploying your decentralized application as a smart contract on a shared blockchain has many limitations. Your smart contract has to share computational resources with every other application, so scalability is limited.

Plus, you’re restricted to the execution environment that shared blockchain uses, so developer flexibility is limited

basically scalability and customizability

Reply:
i hear you 100%. in order to have a place to deploy your contract, though, you'd have to start your own rollup and maintain it. i just am not sure it's the best selling point ATM. there's a lot of places where you can go and easily deploy a smart contract to EVM/SVM compatible chains, etc. however, I think in a very robust case (exchange, game), it would make sense?

Reply:
you wouldn't deploy a smart contract. instead of deploying your dapp as a smart contract, you can deploy it as a rollup so that you want face the scalability and customizability problems. Does it make sense?

Reply:
yea, so maybe we shouldn't say "smart contract"?

Reply:
yeah, i'm gonna sleep on this and read through the whole page again tmrw 😄

Reply:
i'm just not positive it solves challenges during deployment of a contract, as much as it would solve challenges of deploying a L1

Reply:
you're 100% right. it solves more challenges of deploying an L1 than a smart contract. That's why I'm mentioning both. I don't mind removing it if it's confusing.
N.B, this is not the final version, Mustafa will review

Reply:
sounds good!